### ISSUE TITLE
Enhancement Req: WINDOW GUI new buttons for HideMon and SnipeMon

### ISSUE THREAD

**Issue by macinmn** [Necrobot-Private/NecroBot#1145]
While looking at the local map in WINDOW GUI, I see the pokemon nearby local pokestops and would find these two features very useful when one is selected, and I don't believe they have been requested before.

Hide:  particularly when a common monster is on top of other monsters at the same stop, can't be sure what is behind it.  Either hide that specific monster, or hide all of the same species.  I'm thinking you could also allow the option to hide for just the session, or make it permanent (which could always be removed from the config.json later).

Snipe:  it would be great if from the map it could pass and add-to-snipe the location and species data directly via the some method the add-manual-location sniping tool uses.  Right now I take a screen clip, then type it in manually, which isn't necessarily a problem, but I type incorrect numbers or mix up the drop-down species list sometimes and this would be even faster and more accurate.

As always, Thanks for your continued support of the bot and development of fantastic new features!!!

**Comment by hardknoxni69a** [comment]
The Pokemon Displayed On The Map Section Just Click On The Pokemon Then You Can Simply Select Catch  This Pokemon And You Will Go Catch It No Reason To Make Your Life Difficult And Teleporting To A Pokemon That You Will Walk To And Catch Without Raising Suspicion.

**Comment by macinmn** [comment]
I would agree with you if the bot traveled directly there and did not catch or spin on the way, and put it as a priority next destination instead of adding it to the back of the path queue.  If it could follow the humanwalk snipe rules, where it would even speed up, that would be of a big help.  Right now, as I said I have to input coordinates manually to ensure it is caught, otherwise it could be 10-20min before the bot arrives, and it is has disappeared in that time.
